examples/chatterbox.js

```diff
@@ -12,7 +12,7 @@ bot.on('login', function() {
   console.log("settings", bot.settings);
 });
 bot.on('time', function() {
-  console.log("current time", bot.time);
+  //console.log("current time", bot.time);
```

Comment:
Actually this is intentional, this display one line every ~200ms in the console, making everything else the chatterbox example display unreadable. You can see several other things are commented in that example for the same reason ;) (not by me)